src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/JsonSerializerWrapper.Reflection.cs

```diff
@@ -13,6 +15,13 @@ namespace System.Text.Json.Serialization.Tests
 {
     public abstract partial class JsonSerializerWrapper
     {
+        // Ensure that the reflection-based serializer testing abstraction roots KeyValuePair<,>
+        // which is required by many tests in the reflection test suite.
+        [DynamicDependency(DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.PublicProperties, typeof(KeyValuePair<,>))]
+        public JsonSerializerWrapper()
```

Comment:
(nit) - the class is abstract.

```suggestion
        protected JsonSerializerWrapper()
```